etcd3/client_facade.py

```diff
@@ -0,0 +1,49 @@
+from etcd3 import client, exceptions
```

Comment:
I am not statisfied with this solution. However, I have no idea on how to pack this logic into the actual client. Base Problem is, that I don't know how to remeber the invoked function (e.g. put) in order to retry. 
Ideas welcome!